frontend/src/app/libs/user-profile-page/model/user-dto.ts

```diff
@@ -0,0 +1,11 @@
+export interface UserDto {
```

Comment:
seems like this interface already exists in application, please use existing one

Reply:
nope

Reply:
there is no interface to update changed user fields